### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by RaunyGL** [RaunyGL/hackaween-equipe-saw#4] [General]
![Re-food](https://github.com/user-attachments/assets/7c84631c-b1bf-436c-8a0c-b4701189a12e)

**Comment by Lusqinha** [comment]
![378109517-7c84631c-b1bf-436c-8a0c-b4701189a12e-removebg-preview](https://github.com/user-attachments/assets/bc779e7c-bd5b-42ea-8ea9-6a18c1886b9d)

**Comment by thesckurtt** [comment]
![SELO CORRETO 01](https://github.com/user-attachments/assets/4a703544-cb4c-4b78-8a77-9ea269940308)
![SELO CORRETI0 02](https://github.com/user-attachments/assets/6ab31c01-651b-4d75-aa7f-d038439865ed)
![SELO CORRETO 03](https://github.com/user-attachments/assets/c259b87f-f61a-4789-86a3-b22730787468)
